app/src/main/java/de/westnordost/streetcomplete/quests/opening_hours/model/TimeRange.kt

```diff
@@ -7,7 +7,7 @@ import java.util.Locale
 
 /** A time range from [start,end). The times are specified in minutes. */
 @Serializable
-data class TimeRange(val start: Int, val end: Int, val isOpenEnded: Boolean = false) : Comparable<TimeRange> {
+data class TimeRange(val start: Int, val end: Int = UNDEFINED_TIME, val isOpenEnded: Boolean = false) : Comparable<TimeRange> {
```

Comment:
This is how time points are defined in TimeSpan of ch.poole.openinghoursparser. I think it makes sense to adopt the same here because a lot of code is based on that.